phpcs-rulesets/plugin-review.xml

```diff
@@ -46,6 +46,12 @@
 		<type>warning</type>
 	</rule>
 
+	<!-- Sanitized Input rules -->
+	<rule ref="WordPress.Security.ValidatedSanitizedInput">
+		<type>error</type>
+		<severity>7</severity>
+	</rule>
```

Comment:
When we add `ValidatedSanitizedInput` to this file here, my understanding is that the `Plugin_Review_PHPCS_Check` check will run it.

So there's no need for `Sanitized_Input_Check`, as `Plugin_Review_PHPCS_Check` already scans for this. Or am I missing something?

Reply:
Agree. Just adding these rules in the ruleset should trigger the errors.